doc/conf.py

```diff
@@ -56,26 +50,17 @@
 
 # General information about the project.
 project = 'dbxincluder'
-copyright = '2015, Thomas Schraitle'
-author = 'Thomas Schraitle'
+copyright = '2016, Fabian Vogt'
+author = 'Fabian Vogt'
```

Comment:
> Ok, but only if that means that you won't complain about bugs assigned to you :P

They are assigned anyway to me when you leave our department. ;-)

> What's the usual convention for referring to multiple authors?

There are different ways how you can do that. You can add me into a (new) `contributing.rst` file and include it into the main documentation. Ok? :)